helpers/parameter_generator.py

```diff
@@ -123,7 +130,10 @@ def get_alias(infos):
     return pairs
 
 
-def parse_check(check, reverse=False):
+def parse_check(
+    check: Optional[str],
```

Comment:
```suggestion
    check: str,
```

The type `Optional` means that a variable can take the value `None`. You can see that in the `typing` documentation, https://docs.python.org/3/library/typing.html#typing.Union.

> You can use `Optional[X]` as a shorthand for `Union[X, None]`.

`check` in this function cannot take the value `None`, so `Optional` should be removed.